pkg/oauth/server.go

```diff
@@ -245,20 +245,23 @@ func (s *server) RegisterRoutes(server *web.Server) {
 
 	api := group.Group("/api", middleware.CSRF())
 	api.POST("/auth/login", s.Login)
-	api.POST("/auth/logout", s.Logout, s.requireLogin)
+	api.POST("/auth/logout", s.Logout)
```

Comment:
Won't this cause trouble in `s.Logout` for users that are not logged in?

Reply:
Indeed. This is a leftover from before https://github.com/TheThingsNetwork/lorawan-stack/pull/2148#discussion_r394793389